platformio.ini

```diff
@@ -23,23 +23,23 @@ monitor_speed = 115200
 platform = espressif8266@4.2.1
 board_build.f_cpu = 160000000L
 build_type = release
+build_flags = -Os
```

Comment:
If we're thinking about this, I would also suggest increasing the flash speed to 80 MHz (qio is not supported).

Reply:
I was thinking that -Os is the default, isn't it? So that this is actually nfc and just there to have a build_flags entry here ....

Reply:
My bad :) I read as 'O2'

I would say O2 is one thing that can be explored (ELRS does this) https://github.com/ExpressLRS/ExpressLRS/blob/master/src/targets/common.ini#L122